core/src/main/test/com/es/core/model/phone/JdbcPhoneDaoTest.java

```diff
@@ -0,0 +1,94 @@
+package com.es.core.model.phone;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.datasource.DriverManagerDataSource;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.math.BigDecimal;
+import java.util.Date;
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration(locations = "/context/applicationContext-core.xml")
+public class JdbcPhoneDaoTest {
```

Comment:
Please fully rework this class, check implementation and comments https://github.com/DONTWARRYIMAFIME/phoneshop/pull/1